webapp/app/assessment/reducer.ts

```diff
@@ -1,17 +1,17 @@
 import { exportReducer } from '@webapp/utils/reduxUtils'
 
 import * as AssessmentState from '@webapp/app/assessment/assessmentState'
-import ActionTypes from '@webapp/store/app/actions/actionTypes'
 
 import { assessmentLockUpdate, assessmentSectionPropUpdate } from '@webapp/app/assessment/actions'
 import {
   assessmentSectionDataGeneratingValuesUpdate,
   assessmentSectionDataUpdate,
 } from '@webapp/components/Assessment/DataTable/actions'
+import { AppActions } from '@webapp/store'
```

Comment:
this better to be 
```
import { AppActions } from '@webapp/store/app'
```
Let's not export all slices from the root store. better to have each slice exported from its own folder